fundamental-concepts/data-structures-stdlib/1_records.md

```diff
@@ -0,0 +1,87 @@
+One of the most basic data types in DAML is the record. Here's an example:
+
+<pre>
+data Address = Address {
```

Comment:
I think we should teach `with` syntax consistently.

Reply:
Besides I don't like with syntax too much, `with` syntax with nested records becomes very confusing. I would, therefore, show people both ways.